packages/ng/plg-push/plg-push.component.ts

```diff
@@ -1,15 +1,37 @@
-import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { booleanAttribute, ChangeDetectionStrategy, Component, EventEmitter, HostBinding, Input, Output, ViewEncapsulation } from '@angular/core';
+import { ButtonComponent } from '@lucca-front/ng/button';
+import { getIntl } from '@lucca-front/ng/core';
 import { IconComponent } from '@lucca-front/ng/icon';
+import { LU_PLG_PUSH_TRANSLATIONS } from './plg-push.translate';
 
 @Component({
 	selector: 'lu-plg-push',
 	standalone: true,
-	imports: [IconComponent],
+	imports: [IconComponent, ButtonComponent],
 	templateUrl: './plg-push.component.html',
 	styleUrl: './plg-push.component.scss',
 	changeDetection: ChangeDetectionStrategy.OnPush,
 	encapsulation: ViewEncapsulation.None,
 })
 export class PLGPushComponent {
+	protected intl = getIntl(LU_PLG_PUSH_TRANSLATIONS);
+
 	@Input() public heading = '';
+
+	@Input({ transform: booleanAttribute })
+	removable = false;
+
+	@Input({ transform: booleanAttribute })
+	/**
+	 * Is the callout removed? Works with two way binding too.
+	 */
+	removed = false;
```

Comment:
This should be a `model` imo, because we want to get it as input but also changed (same for removedChange, might want to _remove_ it) and having it as `booleanAttribute` makes no sense considering the model approach.